File: 20210719/전자레인지/전자레인지.py
countList = [0, 0, 0]

buttonList = [300, 60, 10]

time = int(input())
for idx, button in enumerate(buttonList):
    if button <= time:
        countList[idx] = time // button
        time = time % button

if time == 0:
    for i in countList:
        print(i, end=" ")
else:
    print(-1)

File: 20210719/전자레인지/전자레인지.js
function solution(time) {
  let countList = [0, 0, 0];
  let buttonList = [300, 60, 10];

  for (let i = 0; i < buttonList.length; i++) {
    if (buttonList[i] <= time) {
      countList[i] = parseInt(time / buttonList[i]);
      time = time % buttonList[i];
    }
  }

  if (time === 0) {
    console.log(countList);
  } else {
    console.log(-1);
  }
}
